requirements-selftests.txt

```diff
@@ -6,18 +6,18 @@ Sphinx>=1.3b1
 # inspektor (static and style checks)
 inspektor>=0.4.5
 # mock (some unittests use it)
-mock>=1.0.0
+mock>=1.0.0; python_version <= '2.7'
```

Comment:
I know python devs claimed 2.7 will be the last version of python 2, but wouldn't it be better still safer to use `python_version < '3'`?